### PR TITLE
Apply useSwr to FileBrowserLite and BundleDetail for previewing bundles

### LINKED ISSUE #3203: Issues with previewing bundle contents
When I preview bundle contents of this bundle (https://worksheets-dev.codalab.org/bundles/0x92096a763a2c4eee95e7877220d31993), notice two things:

image

1. The requests take 10-20 seconds just to preview the bundles, even though this shouldn't really take that long.
2. Two requests are made, when only one request needs to be made.

Here are the bundle contents:

image

### PR DESCRIPTION
**Pull request by yuqijin**
### Reasons for making this change

To avoid duplicate fetch request

### Related issues

fixes #3203 

### Screenshots

Screen Shot 2021-04-20 at 11 42 44 PM

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx [hunk 1]
```diff
@@ -1,250 +1,215 @@
...
+            }
+            );
+
+    const urlContents =
+        '/rest/bundles/' + uuid + '/contents/info/' +  "?" + new URLSearchParams({depth: 1});
+
+    useSWR(urlContents, fetcherContents, {
+        revalidateOnMount: true,
+        refreshInterval:refreshInterval,
+        onSuccess: (response, key, config) => {
```
Comment (epicfaace) [inline frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx:170]: Can we refactor this into a separate function so it's cleaner?

Comment (yuqijin) [reply frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx:170]: Do you mean refactor the onSuccess callback function?

Comment (epicfaace) [reply frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx:170]: Yeah

Comment (yuqijin) [reply frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx:170]: solved :)

**Review by epicfaace — APPROVED** [review]

**Pull request merged**